public/scripts/renderMapToDrawPolygon.js

```diff
@@ -156,10 +156,42 @@ function calculatePolygonFromCoordinates() {
   }
 }
 
+function drawPolygonFromCoordinates(coordinates) {
+  if (coordinates) {
```

Comment:
also check the coordinates are array. i see the default value is assigned as empty object at getGeographyTabData.ts